vue-frontend/src/components/contact/ContactForm.vue

```diff
@@ -318,18 +342,33 @@ export default {
   },
   inject: ["mixpanel"],
   methods: {
+    validateEmailAddress() {
+      var emailRegx =
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+      if (!emailRegx.test(this.email)) {
+        this.showEmailValidationError = true;
+      } else {
+        this.showEmailValidationError = false;
+      }
+    },
     submitForm() {
+      this.showNameValidationError = this.name === "";
```

Comment:
I think you can create separate method like `contactFooterForm` will be more better